deploy/Dockerfile

```
@@ -21,11 +30,9 @@ ADD otel-config-template.yaml /etc/otelcol/config-template.yaml
 # ---
 # Run
 # ---
-FROM grafana/k6:0.45.0
+FROM loadimpact/k6:0.45.0
```

```suggestion
FROM grafana/k6:0.45.0
```
Include the `loadimpact` to `grafana` change